### ISSUE TITLE
[gen3ou] ValueError: spd is not a valid boost

### ISSUE THREAD

**Issue by Viitrexx** [pmariglia/showdown#77]
```
Traceback (most recent call last):
  File "run.py", line 111, in <module>
    asyncio.get_event_loop().run_until_complete(showdown())
  File "D:\Documenten\Python\Python3\lib\asyncio\base_events.py", line 568, in run_until_complete
    return future.result()
  File "run.py", line 94, in showdown
    winner = await pokemon_battle(ps_websocket_client, config.pokemon_mode)
  File "D:\Downloads\Showdown AI 2\showdown\run_battle.py", line 179, in pokemon_battle
    battle = await start_battle(ps_websocket_client, pokemon_battle_type)
  File "D:\Downloads\Showdown AI 2\showdown\run_battle.py", line 170, in start_battle
    battle = await start_standard_battle(ps_websocket_client, pokemon_battle_type)
  File "D:\Downloads\Showdown AI 2\showdown\run_battle.py", line 135, in start_standard_battle
    await read_messages_until_first_pokemon_is_seen(ps_websocket_client, battle, opponent_id, user_json)
  File "D:\Downloads\Showdown AI 2\showdown\run_battle.py", line 113, in read_messages_until_first_pokemon_is_seen
    best_move = await async_pick_move(battle)
  File "D:\Downloads\Showdown AI 2\showdown\run_battle.py", line 38, in async_pick_move
    pool, battle_copy.find_best_move
  File "D:\Documenten\Python\Python3\lib\concurrent\futures\thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "D:\Downloads\Showdown AI 2\showdown\battle_bots\safest\main.py", line 49, in find_best_move
    safest_move = pick_safest_move_from_battles(battles)
  File "D:\Downloads\Showdown AI 2\showdown\battle_bots\safest\main.py", line 32, in pick_safest_move_from_battles
    scores = get_payoff_matrix(mutator, user_options, opponent_options, depth = config.search_depth, prune=True)
  File "D:\Downloads\Showdown AI 2\showdown\engine\select_best_move.py", line 102, in get_payoff_matrix
    state_instructions = get_all_state_instructions(mutator, user_move, opponent_move)
  File "D:\Downloads\Showdown AI 2\showdown\engine\find_state_instructions.py", line 462, in get_all_state_instructions
    all_instructions += get_state_instructions_from_move(mutator, opponent_move, user_move, constants.OPPONENT, constants.SELF, False, instruction)
  File "D:\Downloads\Showdown AI 2\showdown\engine\find_state_instructions.py", line 387, in get_state_instructions_from_move
    temp_instructions += instruction_generator.get_states_from_boosts(mutator, boosts_target, boosts, boosts_chance, instruction_set)
  File "D:\Downloads\Showdown AI 2\showdown\engine\instruction_generator.py", line 743, in get_states_from_boosts
    pkmn_boost = get_boost_from_boost_string(side, k)
  File "D:\Downloads\Showdown AI 2\showdown\engine\instruction_generator.py", line 1242, in get_boost_from_boost_string
    raise ValueError("{} is not a valid boost".format(boost_string))
ValueError: spd is not a valid boost
```

**Comment by pmariglia** [comment]
Thanks for opening this, looks like a bug.

I'll take a closer look when I get a chance.

**Comment by pmariglia** [comment]
Thanks again for bringing this up. Had some incorrect boost strings in the gen3 move modifier JSON. The bot must've been trying to use `crunch`.

Fixed in daddd36c4657659f429db671d0f572ac787ecff5

**Comment by pmariglia** [comment]
Closing this issue. Let me know if you still have a problem!
